src/pages-and-resources/proctoring/Settings.test.jsx

```diff
@@ -463,8 +463,10 @@ describe('ProctoredExamSettings', () => {
     });
 
    it('Does not include lti_external as a selectable option', async () => {
-      const courseData = mockGetFutureCourseData;
-      courseData.available_proctoring_providers = ['lti_external', 'proctortrack', 'mockproc'];
+      const courseData = {
```

Comment:
What was happening here? Was it sharing the same object across tests?

Reply:
yeah exactly, this intent was to copy it not modify it.